photon-client/src/main/scala/com/linkedin/photon/ml/data/avro/AvroDataReader.scala

```diff
@@ -271,7 +265,7 @@ object AvroDataReader {
     fieldNames
       .toSeq
       .flatMap { fieldName => Option(record.get(fieldName)) match {
-        case Some(recordList: JList[_]) => recordList.asScala.toSeq
+        case Some(recordList: JList[_]) => recordList.asScala.toSeq // this .toSeq intentional
```

Comment:
Well, here - it would be appropriate. I don't feel like inserting a line for the comment. This comment was just supposed to be helpful because the IDE is complaining about that `toSeq`. I'll remove it.